pkgs/applications/science/logic/abc/default.nix

```
@@ -4,13 +4,13 @@
 
 stdenv.mkDerivation rec {
   pname   = "abc-verifier";
-  version = "2022.07.27";
+  version = "2022.09.08";
```

```suggestion
  version = "unstable-2022-09-08";
```
there are not tags or mentions of this in the git repo.

---

Thanks for noticing.